index.html

```diff
@@ -75,7 +75,9 @@ <h1 class="info-title">Comodidades</h1>
           <div class="pointer-list__oval pointer-list__oval--gray" id="divBtn3"></div>
           <div class="pointer-list__oval pointer-list__oval--gray" id="divBtn4"></div>
         </div>
-        <button href="#" class="button button--medium button--secondary">Ver galeria</button>
+        <a href="#">
+          <button class="button button--sm button--lg-desktop button--secondary">Ver galeria</button>
```

Comment:
`button` tag should not go inside an `a` tag, you could just use the button tag, and put `onClick` attribute to redirect to the gallery page. You can research on how to do this, remember that button should not have an `href` attribute.

Reply:
done